software/earl/http-api.go

```diff
@@ -7,6 +7,35 @@ import (
 	"sort"
 	"sync"
 	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func secondsDurationBuckets() []float64 {
+	return []float64{
+		0.005, /* 5ms */
```

Comment:
I suspect that we want also a `1ms` bucket.

Reply:
Ehh, anything faster than 5ms is good enough ;-)